src/main/scala/io/iohk/ethereum/nodebuilder/StdNode.scala

```diff
@@ -62,6 +62,7 @@ abstract class BaseNode extends Node {
 
     startPeerManager()
 
+    startPortForwarding
```

Comment:
Since this is side effecting, it should have `()` appended to it. I'd also recommend adding some logging somewhere so allocation can't just silently fail.

Reply:
With the `AtomicReference` this is also fixable :+1: